### ISSUE TITLE
Add pennant control

### ISSUE THREAD

**Issue by DougCoffland** [buildbotics/bbctrl-firmware#152]
I've had a couple of customer requests to allow adding pennant controls. Here's an example of something that we could offer. We could sell it as an accessory to people that are not satisfied with the game controller.

https://www.amazon.com/HandWheel-Optical-Encoder-Generator-Pendant/dp/B00D7VSR6I#customerReviews

**Comment by jcoffland** [comment]
Yeah, that pendant has been on my Amazon list for at least a year now.  I think it would be pretty easy.
